beacon-chain/rpc/apimiddleware/custom_hooks.go

```diff
@@ -397,6 +398,30 @@ type tempSyncSubcommitteeValidatorsJson struct {
 	Validators []string `json:"validators"`
 }
 
+type tempLightClientUpdatesByRangeJson struct {
+	Updates []*LightClientUpdateJson `json:"updates"`
+}
+
+// https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getLightClientUpdatesByRange
+// grpc-gateway returns a struct with slice embedded, but spec says it should be a list directly instead of a struct.
+func prepareLightClientUpdates(body []byte, responseContainer interface{}) (apimiddleware.RunDefault, apimiddleware.ErrorJson) {
```

Comment:
gRPC doesn't support return an array directly (have to embed the array inside a response object)

REST spec defines the return type is an array.

Therefore this special hook function is added and doing the conversion.